sonar-dotnet-shared-library/src/main/java/org/sonarsource/dotnet/shared/plugins/GeneratedFileFilter.java

```diff
@@ -39,7 +42,7 @@
   private final ProtobufImporters protobufImporters;
 
   private boolean initialized;
-  private Set<String> generatedFilePaths;
+  private Set<Path> generatedFilePaths = new HashSet<>();

```

Comment:
The change from Path to String broke the filter, I am returning Path back as the recommended way to work with paths.